modules/management-console-agent/src/test/java/org/apache/ignite/internal/agent/action/controller/QueryActionsControllerTest.java

```diff
@@ -388,6 +391,62 @@ public void shouldCancelLongQuery() {
         );
     }
 
+    /**
+     * 1. Execute a query with sleep function.
+     * 2. Send kill query action with global query id for a query from 1.
+     * 3. Assert that action was completed and no one running queries exists in a cluster.
+     */
+    @Test
+    public void shouldKillRunningQuery() {
```

Comment:
Please add a test that kill a query started from Ignite directly without using GGCC API.

Reply:
Added